helm-charts/flink-historyserver/templates/efs.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: efs-flink-history
+parameters:
+  provisioningMode: efs-ap
+  fileSystemId: "{{- .Values.aws_efs_file_system_id }}"
+provisioner: "efs.csi.aws.com"
+---
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: flink-historyserver-efs-pv
```

Comment:
I like that this has 'historyserver' in the name, so it gets used specifically just for this and not much more :)

Reply:
@ranchodeluxe I think this still needs to be fixed?